packages/remix-dev/devServer_unstable/index.ts

```diff
@@ -107,7 +107,8 @@ export let serve = async (
     console.log(`> ${cmd}`);
     let newAppServer = execa
       .command(cmd, {
-        stdio: "pipe",
+        stdio: ['ignore', 'inherit', 'inherit'],
```

Comment:
Pretty sure we need `pipe` so that we can check for app ready messages in stdout, right @jacob-ebey ?

Reply:
Actually, I think we can use a [similar approach as our HMR tests](https://github.com/remix-run/remix/blob/main/integration/hmr-test.ts#L246-L250) to inspect `stdout` without needing `pipe`s for `execa` at all. @jacob-ebey let's chat about this when I'm back